freeipa/src/main/java/com/sequenceiq/freeipa/converter/stack/StackToDescribeFreeIpaResponseConverter.java

```diff
@@ -123,4 +125,8 @@ private PlacementResponse convert(Stack source) {
         placementResponse.setRegion(source.getRegion());
         return placementResponse;
     }
+
+    private Boolean convertAvailabilityStatus(StackStatus status) {
+        return status == null ? Boolean.FALSE : status.getDetailedStackStatus().isAvailable();
+    }
```

Comment:
Do you want to log the detailedStackStatus to know what status exactly was converted to available/unavailable? So if it is unavailable, we know what the real status is. Or if you can search easily now in log already, then ignore.

Reply:
No, because the detailed status is also in the converted response.